build_tools/mako/configuration.py

```diff
@@ -176,7 +176,10 @@ def get_s20_default_target_list(skipped_target=None, batch_config=None):
               "--iree-vulkan-target-triple=valhall-g77-unknown-android10",
               "--iree-flow-dispatch-linalg-on-tensors",
               "--iree-codegen-spirv-experimental-linalg-on-tensors",
-              "-iree-flow-inline-constants-max-byte-length=2048"
+              "-iree-flow-inline-constants-max-byte-length=2048",
+              "-iree-spirv-enable-vectorization",
```

Comment:
Actually this is not needed given https://github.com/google/iree/pull/5280. But good to keep it here for clarity.

Reply:
got you, thanks!